cirq-ionq/cirq_ionq/ionq_devices.py

```diff
@@ -103,19 +101,19 @@ def _decompose_single_qubit(self, operation: cirq.Operation) -> cirq.OP_TREE:
             yield gate(qubit)
 
     def _decompose_two_qubit(self, operation: cirq.Operation) -> cirq.OP_TREE:
-        """Decomposes a two qubit gate into XXPow, YYPow, and ZZPow plus single qubit gates."""
+        """Decomposes a two qubit unitary Rz,Rx,and CNOT."""
```

Comment:
This is a pretty big change and I'm surprised that none of the existing tests fail after this change. 

Please add tests which would pass now but would have failed if the change is reverted.

Reply:
The test should also be useful to guard against any potential change to the default decomposition of `PhasedXPowGate` into `Rx` / `Rz` gates.

Reply:
Makes sense! I added a check that the valid gateset is used, in addition to the unitary equivalence check being done. WDYT?